### PR TITLE
Biweekly Updates - Jul 22 (Update v2-release-notes.md)

### PR DESCRIPTION
**Pull request by heyitswei**
Release notes for Enterprise and Proton

### CODE REVIEW

File: docs/v2-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 22, 2024
+
+### Timeplus Proton v1.5.14
```
Comment (jovezhong) [inline docs/v2-release-notes.md:7]: missed Timeplus Proton v1.5.13
* fix: properly clear the format schema cache when the format schema is dropped
* fix: a null pointer access before type cast

File: docs/v2-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+  * New feature: Remote user-defined functions (UDFs) are now supported.
```
Comment (jovezhong) [inline docs/v2-release-notes.md:8]: We support remote UDF long time ago. In 1.5.14, the user can create remote UDF via SQL (previously via REST API, or Timeplus Enterprise user creates it via UI)

File: docs/v2-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
+    * Example: `CREATE REMOTE FUNCTION ip_lookup(ip string) RETURNS string URL 'https://abc.lambda-url.us-west-2.on.aws/'`
+
```
Comment (jovezhong) [inline docs/v2-release-notes.md:10]: timeplus-native-jdbc v2.0.2 (https://github.com/timeplus-io/timeplus-native-jdbc/releases/tag/v2.0.2)
* support bigint and decimal type 
* Bugfix: invalid version for lowcardinality keys

File: docs/v2-release-notes.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
...
+### Timeplus Enterprise
+  * In the SQL Console, see a query's pipeline after running a query. Note: This is available for single-node on-prem deployments.
+  * New stream mode: mutable streams, where values with the same primary key(s) will be overwritten. More advanced configuration options will be available soon.
```
Comment (jovezhong) [inline docs/v2-release-notes.md:13]: * New external stream wizard for AutoMQ

Comment (jovezhong) [reply docs/v2-release-notes.md:13]: missing many other features.

* timeplusd
  * feat: [new mutable stream](mutable-stream) for fast UPSERT and high performance point or range query.
  * perf: better asof join performance
  * feat: [parallel key space scan](mutable-stream#key_space_full_scan_threads)
  * feat: force_full_scan for mutable stream
  * feat: user management on cluster
  * fix: [alter stream .. modify settings ..](sql-alter-stream#stream_ttl)
  * feat: support remote UDF on cluster
  * feat: primary key columns in secondary key
  * feat: _tp_message_key to [read/write message keys in Kafka](proton-kafka#messagekey)
  * feat: [Kafka schema registry support for Avro output format](proton-schema-registry#write)
  * feat: collect node free memory usage. You can get it via `select cluster_id, node_id, os_memory_total_mb, os_memory_free_mb, memory_used_mb, disk_total_mb, disk_free_mb, timestamp from system.cluster`
* timeplus_appserver
  * feat: added mutable stream support
  * feat: more metrics for mv stats
  * feat: stream and external stream metrics
  * feat: adapted timeplusd cluster metrics
  * feat: remove ably source
  * feat: improve stats on data lineage
* timeplus_web
  * feat: show more resources stats
  * feat: update license UI
  * feat: show detailed version and build time for components
  * feat: UI for mutable stream CRUD
  * feat: show throughput on data lineage
  * feat: refined GetStarted UI on home page
  * feat: support multinode stats on data lineage
* cli
  * feat: new command for backup/restore data and configuration
  * feat: new command for synchronizing resources to timeplusd
  * feat: when you start Timeplus Enterprise for the first time, auto-create a dashboard to show usage and stats. The template resides in `timeplus/bin/.dashboard`

**Review by jovezhong — CHANGES_REQUESTED** [review]

File: docs/v2-release-notes.md (hunk 4 above)
Comment (heyitswei) [reply docs/v2-release-notes.md:13]: Hi @jovezhong, is autoMQ ready to be shared now?

Comment (jovezhong) [reply docs/v2-release-notes.md:18]: putting in docs release notes is okay. Not decided yet whether to put a English blog for AutoMQ

File: docs/v2-release-notes.md [hunk 5]
```diff
@@ -5,8 +5,13 @@ This page summarizes changes for each major update in Timeplus Enterprise v2 and
 ## Jul 22, 2024
 
 ### Timeplus Proton v1.5.14
-  * New feature: Remote user-defined functions (UDFs) are now supported.
+  * Remote user-defined functions (UDFs) can now be created via SQL.
     * Example: `CREATE REMOTE FUNCTION ip_lookup(ip string) RETURNS string URL 'https://abc.lambda-url.us-west-2.on.aws/'`
+  * Timeplus-native-jdbc is updated (v2.0.2):
```
Comment (jovezhong) [inline docs/v2-release-notes.md:10]: Timeplus-native-jdbc is a different repo, not part of Timeplus Proton or Timeplus Enterprise. If we have to chose one, it belongs to Timeplus Enterprise.. Should not be put between Proton 1.5.13 and 1.5.14

**Review by jovezhong — CHANGES_REQUESTED** [review]

**Review by jovezhong — APPROVED** [review]

**Pull request merged**